src/cloud-api-adaptor/hack/build.sh

```diff
@@ -59,10 +59,13 @@ function build_caa_payload_image() {
 		--build-arg COMMIT="${commit}" \
 		--build-arg YQ_VERSION="${YQ_VERSION}" \
 		--build-arg YQ_CHECKSUM="${YQ_CHECKSUM}" \
+		--load \
 		-f cloud-api-adaptor/Dockerfile \
 		${tag_string} \
-		--push \
 		.
+
+    docker push  ${registry}/${name}
+
```

Comment:
Hi @davidhadas, do you mind explaining *why* this change is needed with your commit message, please?

Reply:
Sure. I hope it is now clearer ;)

Reply:
@davidhadas I mean in the git message, not the PR description. PR description does not get merged.

Reply:
@beraldoleal done.